include/pros/motors.hpp

```diff
@@ -71,10 +70,17 @@ class Motor {
 	explicit Motor(const std::uint8_t port, const motor_gearset_e_t gearset, const bool reverse,
 	               const motor_encoder_units_e_t encoder_units);
 
+	explicit Motor(const std::uint8_t port, const motor_gear_e_t gearset, const bool reverse,
+	               const motor_encoder_units_e_t encoder_units);
+
 	explicit Motor(const std::uint8_t port, const motor_gearset_e_t gearset, const bool reverse);
 
+	explicit Motor(const std::uint8_t port, const motor_gear_e_t gearset, const bool reverse);
+
 	explicit Motor(const std::uint8_t port, const motor_gearset_e_t gearset);
 
+	explicit Motor(const std::uint8_t port, const motor_gear_e_t gearset);
+
 	explicit Motor(const std::uint8_t port, const bool reverse);
```

Comment:
I think we have a misunderstanding as to what the "overload" means. We should have the option to pass in a pros::Color that is red to red gear cartridge, blue to blue gear cartridge, and green to green gear cartridge.